lib/gnat/command.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Gnat.Command do
+  @newline "\r\n"
+  @unsub "UNSUB"
+
+  def build(:unsub, sid, []), do: [@unsub, " #{sid}", @newline]
+  def build(:unsub, sid, [max_messages: max]), do: [@unsub, " #{sid}", " #{max}", @newline]
```

Comment:
The matching in method signatures is pretty awesome! I like having the commands in a common place.